### ISSUE TITLE
Opening new notebook in VSCode defaults to python kernel

### ISSUE THREAD

**Issue by bisen2** [dotnet/interactive#1189]
### Describe the bug
When opening a new dotnet interactive notebook in VSCode using the `.NET Interactive: Create new blank notebook` from the command palette, the resulting notebook uses the python jupyter kernel. The notebook can be reopened with the .NET kernel by running `View: Reopen Editor With...` and selecting `.NET Interactive for Jupyter`, but it initially opens with the python kernel. I imagine this is what caused the confusion in #1188.

### Please complete the following:

**Which version of .NET Interactive are you using?**

* .NET Interactive Version: 1.0.217503+a53405c9aae40b1ff468e38b641a981f615d8fdb
* VSCode Insiders Version: 1.55.0-insiders
* Jupyter Extension Version: v2021.5.685944597

- OS
    - [x] Windows 10
    - [ ] macOS
    - [ ] Linux (Please specify distro)
    - [ ] iOS
    - [ ] Android
 - Browser 
    - [ ] Chrome
    - [ ] Edge
    - [ ] Firefox
    - [ ] Safari
 - Frontend
    - [ ] Jupyter Notebook
    - [ ] Jupyter Lab
    - [ ] nteract
    - [ ] Visual Studio Code 
    - [x] Visual Studio Code Insiders
    - [ ] Other (please specify)

**Comment by jonsequitur** [comment]
Yeah, this is the same issue. It's worth adding that currently the choice you make is sticky, but it's not obvious that that's the case. And it seems like that choice overrides the  `.NET Interactive: Create new blank notebook` command.

**Comment by DonJayamanne** [comment]
FYI - Based on disccussions we had with PMs & others, Jupyter extension added an API endpiont for .NET extension to contribute to

`registerNewNotebookContent(options: { defaultCellLanguage: string }): void;`

Not sure if thats not used. 
Also last time we had a sync I mentioned a problem with this approach, we need a static contribution.

**Comment by DonJayamanne** [comment]
I believe this has been resolved in the Insiders version of the Jupyter extension.
Please could you test this and confirm.

**Comment by brettfo** [comment]
The vscode-jupyter issues have all been addressed, and now the only blocking issue is on VS Code via microsoft/vscode#113937.  I've updated the original issue with this link.

**Comment by DonJayamanne** [comment]
I don't think this is true anymore. 
I think this can be closed, I don't see a situation where python will be the preferred/selected kernel (the Jupyter extension will not be displaying python under any circumstances).

Based on my conversation, the problem is that a 3rd party extension could be the preferred kernel (which works be a separate issue all together)
This issue leads one to believe that Jupyter extension (python) kernel is preferred, when it's no longer the case.

**Comment by DonJayamanne** [comment]
@brettfo should this issue be closed.
As it's no longer possible for python to be preferred.i think your concern is that  another extension targeting .net notebooks  could be the preferred kernel in vscode. Ie that's a different issue all together (as the label and issue title leads others to think this is blocked on Jupyter when it isn't) - ie it's not an interop issue with Jupyter.

**Comment by bisen2** [comment]
FWIW I just tested this using the insiders build and it is working as expected. The .NET extension defaults to C# and the Jupyter extension does not.
